doc/operator-backup-and-restore.md

```diff
@@ -0,0 +1,135 @@
+# 3scale installation Backup and Restore using the operator
+
+## Table of contents
```

Comment:
Check TOC anchors, some of them are invalid

Reply:
:+1:

Reply:
I've fixed the ones I've been able to detect broken